### DISCUSSION TITLE
How to only export the mesh plot (no background, no scales)?

### DISCUSSION THREAD

**Discussion by nicklimmm** [pyvista/pyvista#5675] [Q&A]
After running `mesh.plot(...)` with `screenshot='my_img.png'`, it outputs this:
![image](https://github.com/pyvista/pyvista/assets/65026286/22a368bc-f017-454b-befb-0a046bf73bca)

But I would like to get this result:
![image](https://github.com/pyvista/pyvista/assets/65026286/31c6edcf-9d68-40e4-8715-b6435fa43910)

How can I achieve this?

**Comment by akaszynski** [comment]
Try using [camera.tight](https://docs.pyvista.org/version/stable/api/core/_autosummary/pyvista.Camera.tight.html). For example:

```py
import pyvista as pv
from pyvista import examples
puppy = examples.download_puppy()
pl = pv.Plotter(border=True, border_width=5)
_ = pl.add_mesh(puppy, rgb=True)
pl.camera.tight()
pl.show()
```

Also, be sure to disable `show_scalar_bar` within [add_mesh](https://docs.pyvista.org/version/stable/api/plotting/_autosummary/pyvista.Plotter.add_mesh.html).
